### ISSUE TITLE
Errors in Madgwick Orientation Filter

### ISSUE THREAD

**Issue by ychen441** [Mayitzin/ahrs#95]
Hi Mario, there is a mistake in the loss function here in: Madgwick Orientation Filter>Orientation as solution of Gradient Descent. Please fix these three components. Thx :)

<img width="521" alt="error" src="https://user-images.githubusercontent.com/127380024/224546401-cd41e258-096b-49c5-958a-48db691d172b.png">

**Comment by Mayitzin** [comment]
Thanks for bringing this to attention. I corrected it and committed the changes in 97609e0256a32b0af6d08ec7f577a9ca62e6e0d7. The documentation should be corrected now.
